src/org/omegat/filters3/xml/DefaultXMLDialect.java

```diff
@@ -110,7 +110,9 @@ public void defineIntactTags(String[] tags) {
 
     /** The set of defined paragraph tags. */
     private MultiMap<String, String> translatableTagAttributes = new MultiMap<String, String>();
-
+    /** The set of defined translation status tags */
+    private MultiMap<String, String> translationStatusAttributes = new MultiMap<>();
```

Comment:
This doesn't appear to ever be populated? If it's not intended for `DefaultXMLDialect` to have a real implementation of this, then I don't really like it having these dummy maps.